renderer.js

```diff
@@ -1249,6 +1277,17 @@ class SDFGRenderer {
 
         draw_sdfg(this, ctx, g, this.mousepos);
 
+        if (this.box_select_rect) {
+            this.ctx.beginPath();
+            // TODO: change stroke width based on zoom level, i.e. make it wider
+            // when zoomed out, so the selection box is still visible.
+            this.ctx.strokeStyle = 'red';
```

Comment:
dotted black line perhaps?

Reply:
Black performs really poorly against the dark background, visually. Gray seems to be a good compromise and still stands out enough.